komga-webui/src/locales/en.json

```diff
@@ -458,22 +458,22 @@
     },
     "filename_chooser": {
       "button_choose": "Choose",
-      "field_destination_filename": "Destination file name",
```

Comment:
https://en.wikipedia.org/wiki/Filename
This more common variant avoids error in translation.